auth/auth/auth.py

```diff
@@ -553,8 +545,15 @@ async def post_create_user(request, userdata):  # pylint: disable=unused-argumen
 
 
 @routes.get('/api/v1alpha/users')
-@auth.rest_authenticated_developers_only
-async def rest_get_users(request, userdata):  # pylint: disable=unused-argument
+async def rest_get_users(request):
+    session_id = await get_session_id(request)
+    if not session_id:
+        raise web.HTTPUnauthorized()
+    userdata = await get_userinfo(request, session_id)
+    is_developer_or_sa = userdata is not None and (userdata['is_developer'] == 1 or userdata['is_service_account'] == 1)
+    if not is_developer_or_sa:
+        raise web.HTTPUnauthorized()
```

Comment:
Elsewhere we special case CI, can we continue to do that here? I don't want, e.g., my service account to be able to hit this endpoint.

Reply:
Fair point, I've restricted it further to just CI. For future reference though, `is_service_account` doesn't mean it could be a user service account, rather that the account is one of the accounts for our services, like `ci`, `batch`, `grafana`, etc.

Reply:
ah, OK, still better though.